.github/workflows/api-npm-dependency-updates.yaml

```diff
@@ -0,0 +1,48 @@
+name: API NPM Dependency Updates
+
+on:
+  schedule:
+    - cron: '0 8 * * 1' # Every Monday at 8:00am UTC - roughly midnight Pacific Time.
```

Comment:
Updated as per adam's request to align all scheduled reports. Please refer to https://github.com/bcgov/PIMS/blob/dev/.github/workflows/app-npm-dependency-updates.yml

Reply:
Updated